### ISSUE TITLE
Download/upload asynchronously

### ISSUE THREAD

**Issue by dotdeas** [hugsbrugs/php-ftp#1]
Hi,

Love the work but is it possible to make the script work asynchronously, like php's ftp_nb_get? This would be useful for creating output of each file being uploaded or downloaded, or link it to a progressbar function to output some sort of info that its working on the operation and not just return true or false.

**Comment by hugsbrugs** [comment]
Yes that would be great, but not possible because my php-sftp lib based on https://github.com/phpseclib/phpseclib does not support this feature and I need those two libs (ph-ftp and php-sftp) to behave the same way to use them in my apps ... but you can fork it to a php-nb-ftp for example !

**Comment by dotdeas** [comment]
I understand, will make my own fork. Thanks for the great work.
